test/run_speedy_with_output.jl

```diff
@@ -23,12 +23,12 @@ end
 
 @testset "Restart from output file" begin
     
-    p = run_speedy(Float32, initial_conditions=:restart, restart_id=1)
+    p = run_speedy(Float32, initial_conditions=StartFromFile, restart_id=1)
 
     p, d, m = initialize_speedy(Float32, ShallowWater, output=true, run_id="restart-test")
     SpeedyWeather.time_stepping!(p, d, m)
  
-    progn, diagn, model = initialize_speedy(Float32, ShallowWaterModel, initial_conditions=:restart, restart_id="restart-test")
+    progn, diagn, model = initialize_speedy(Float32, ShallowWaterModel, initial_conditions=StartFromFile, restart_id="restart-test")
 
     for varname in propertynames(progn.layers[1].leapfrog[1])
         if SpeedyWeather.has(progn, varname)
```

Comment:
It would suggest to just add the scaling to the test. `p` will always be in model units, and setting the initial conditions in SI units also makes sense.

Reply:
I decided to move all scaling into time_stepping!. This way all prognostic variables outside the dynamical core aren't scaled, and then within one function, before the main time loop starts things are scaled and unscaled afterwards. The netcdf output of course still countains unscaling too. But I hope that makes things clearer. That way the user shouldn't come in contact with scaled prognostic variables and we shield this property better in the dynamical core.